tensorflow/core/ops/ops.pbtxt

```diff
@@ -10313,6 +10313,154 @@ op {
   summary: "Computes gradients of the maxpooling function."
 }
 op {
+  name: "MaxPoolGradGrad"
```

Comment:
Okay. I may drop these changes then. Also, as I noted before, the previous declaration of MaxPool3dGrad didn't use template attribute `T` for types of input and output. So ops compatibility tests will fail. I assume that the correct declaration is to use template attribute.